aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -8,41 +8,33 @@ By [Peter Kellner](http://peterkellner.net)
 
 The ```AnchorTagHelper``` enhances the html anchor (`<a ... ></a>`) tag. A new set of attributes are defined that work with the anchor tag. The link generated (on the href tag) is based on a combination of these new attributes that work together to form the final redirect URL. That URL can include an optional protocol such as https.
 
-For reference, the following ASP.NET ```startup.cs``` and ```SpeakerController.cs``` are defined as you would expect in a default Visual Studio .Net Core Web Project.
+For reference, the following ASP.NET ```SpeakerController.cs``` is defined as you would expect in a default Visual Studio .Net Core Web Project.
 
-![](../_static/ProjectControllers.png)
 
-**Startup.cs**
-```
-// This method gets called by the runtime. 
-// Use this method to configure the 
-// HTTP request pipeline.
-public void Configure(IApplicationBuilder app, 
-    IHostingEnvironment env, 
-    ILoggerFactory loggerFactory)
-{
-    app.UseMvcWithDefaultRoute();
-    //...
-}  
-```
 
 <br/>
 **SpeakerController.cs** 
 
 [!code-csharp[SpeakerController](../sample/TagHelpersBuiltInAspNetCore/src/TagHelpersBuiltInAspNetCore/Controllers/SpeakerController.cs)]
 
-<br/>
-These attributes are defined as follows:
 
-## asp-controller
+## The attributes are defined as follows
+
+- - -
+
+### asp-controller
 
 ```asp-controller``` is used to associate which controller will be used to generate the final URL. The only valid choices are controllers that exist in the current project. In our case, to get to a list of all speakers or speaker details you would specify ```asp-controller="Speaker"```. If you only specify ```asp-controller``` and no ```asp-action``` the URL will generate without an error but will likely not be what you expect.
 
-## asp-action
+- - -
+
+### asp-action
 
 ```asp-action``` is the name of the method in the controller that will be included in the final URL. That is, in the example, if the route to the Speaker Detail page is wanted, then the attribute should be set to ```asp-action=Detail```. You should always set ```asp-controller``` when specifying ```asp-action```. If no ```asp-action``` is specified then a URL will generate without an error but will likely not be what you expect.
 
-## asp-route-
+- - -
+
+### asp-route-
 
 ```asp-route-``` is basically a wild card route prefix. Any value you put after the trailing dash will be interpreted as the parameter to pass into the route. For example, if you create a tag as follows: 
```

Comment:
* "wild card" to "wildcard" in IT contexts
* I'd add either an asterisk (`*`) or a hint for the attribute ... either `asp-route-*` or `asp-route-{value}`
* For `asp-route-id-="11"` below ... no last dash there AFAIK. I think it should be `asp-route-id="11"`. This occurs in a couple of spots here.

Below this line, I see ...
* href should receive backticks
* Codeblocks should break across lines
* The construction of this section seems a bit awkward to me with dangling phrases between code blocks. Personally, I'd clean it up a bit with complete sentences that set off the code examples with colons.
* "html" to "HTML"